CHANGELOG.md

```diff
@@ -1462,6 +1462,10 @@ should use 4.0.1-alpha.0 for testing.
 
 ## [Unreleased]
 
+### Breaking Changes
+
+-   dropped support for NodeJs@14
```

Comment:
With the drop of nodejs 14 support, we should eventually replace the 14 tests for 18